tools/experiments/experiments.js

```diff
@@ -22,6 +22,8 @@ import {getMode} from '../../src/mode';
 import {isExperimentOn, toggleExperiment} from '../../src/experiments';
 import {listenOnce} from '../../src/event-helper';
 import {onDocumentReady} from '../../src/document-ready';
+//TODO(@cramforce): For type. Replace with forward declaration.
+import '../../src/service/timer-impl';
```

Comment:
It makes the compiler happy, because the compilation unit references the type, but not its implementation.

I will soon implement a more general solution to this problem.